docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -5,9 +5,14 @@ This guide contains instructions to upgrade from version v7.0.0-beta6 to Unrelea
 **Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
 
-[Upgrade from v7.0.0-beta6 to Unreleased]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta6...HEAD
-
 ## [shopsys/framework]
 ### Application
+- naming of promo code and discount code was unified ([#844](https://github.com/shopsys/shopsys/pull/844)) 
+    - dump translations using `php phing dump-translations` and fill in the translations based on the changes from pull request
+    - rename occurrences of `discount code` into `promo code` based on the changes from pull request in following files
```

Comment:
Please change order of these instructions. First, they need to rename occurrences and dump translations after that.